### PR TITLE
Adding jenkins and circle ci to repo

### PR DESCRIPTION
**Pull request by JamesAndrews3**

### CODE REVIEW

**Comment by JamesAndrews3** [comment]
Will complete when get back to 251 tomorrow to access Jenkins server.

**Review by biofotis — APPROVED** [review]

**Review by biofotis — APPROVED** [review]
could you do fix the CI errors please?

File: sr_ur_robot_hw/src/sr_ur_control_server.cpp [hunk 1]
```diff
@@ -212,26 +213,34 @@ void UrControlServer::start()
 
   // get the port that the OS assigned
   sockaddr address_with_bound_port;
-  int length = (int)sizeof(sockaddr);
+  int length = static_cast<int>(sizeof(sockaddr));
   status = uv_tcp_getsockname(&server_stream_, &address_with_bound_port, &length);
   ROS_ASSERT(0 == status);
   int reverse_port = ntohs(((struct sockaddr_in*)&address_with_bound_port)->sin_port);
 
+<<<<<<< Updated upstream
```
Comment (biofotis) [inline sr_ur_robot_hw/src/sr_ur_control_server.cpp]: i think that should be removed along with the rest of git staff

**Review by biofotis — DISMISSED** [review]
Have you compiled this? (with reinterpret_cast addition)

**Comment by JamesAndrews3** [comment]
Tested using fh_ur10_hand2.launch and sr_robot_commander.py:
- can set velocity and acceleration
- can move to joint space pose
- can move to cartesian space pose

**Review by AndriyPt — APPROVED** [review]

**Pull request merged**